File: pojo/member.js
function Member()
{
var firstName,lastName,mobileNumber,emailId,password;
this.setFirstName=function(firstName)
{
this.firstName=firstName;
}
this.setLastName=function(lastName)
{
this.lastName=lastName;
}
this.setMobileNumber=function(mobileNumber)
{
this.mobileNumber=mobileNumber;
}
this.setEmailId=function(emailId)
{
this.emailId=emailId;
}
this.setPassword=function(password)
{
this.password=password;
}
this.getFirstName=function()
{
return this.firstName;
}
this.getLastName=function()
{
return this.lastName;
}
this.getMobileNumber=function()
{
return this.mobileNumber;
}
this.getEmailId=function()
{
return this.emailId;
}
this.getPassword=function()
{
return this.password;
}
}

File: WEB-INF/classes/com/thinking/machines/dmodel/services/pojo/Member.java
package com.thinking.machines.dmodel.services.pojo;
public class Member
{
private String emailId;
private String password;
private String firstName;
private String lastName;
private String mobileNumber;
public Member()
{
this.emailId=null;
this.password=null;
this.firstName=null;
this.lastName=null;
this.mobileNumber=null;
}
public void setEmailId(String emailId)
{
this.emailId=emailId;
}
public String getEmailId()
{
return this.emailId;
}
public void setPassword(String password)
{
this.password=password;
}
public String getPassword()
{
return this.password;
}
public void setFirstName(String firstName)
{
this.firstName=firstName;
}
public String getFirstName()
{
return this.firstName;
}
public void setLastName(String lastName)
{
this.lastName=lastName;
}
public String getLastName()
{
return this.lastName;
}
public void setMobileNumber(String mobileNumber)
{
this.mobileNumber=mobileNumber;
}
public String getMobileNumber()
{
return this.mobileNumber;
}
}
